cEP-0020.md

```diff
@@ -0,0 +1,543 @@
+# Newcomer metrics and Gamification
+
+| Metadata |                                                        |
+| -------- | ------------------------------------------------------ |
+| cEP      | 20                                                     |
+| Version  | 1.0                                                    |
+| Title    | Newcomer metrics and Gamification system               |
+| Authors  | Shrikrishna Singh <mailto:krishnasingh.ss30@gmail.com> |
+| Status   | Proposed                                               |
+| Type     | Process                                                |
+
+## Abstract
+
+This cEP describes how a gamification system will automate the
+process of becoming a `Developer` from a `Newcomer`, as a part of
```

Comment:
this needs to be revised slightly now that it wont 100% automate the process of "Developer" (note it is a human language term, not a literal string, so no backticks needed).

Also "Developer" and "Newcomer" are added here, but without any context.  These are roles within the coala community.

Also the focus of this first sentence should be about what are the objectives of the project, which isnt about just the process, but hopefully is about 'the service' we provide to newcomers, which includes technical skills but we also provide encouragement & motivation, and this project is about improving the encouragement & motivation and pathway to learning the technical skills.

Something like...

> This cEP describes a gamification system to automate the recognition of activities performed by people who join the coala community, and encourage them to progress towards obtaining the role of "Developer" in the community.

> This cEP is part of a [GSoC project](..).